graph/src/data/subgraph/schema.rs

```diff
@@ -163,6 +163,14 @@ impl SubgraphHealth {
             SubgraphHealth::Failed => "failed",
         }
     }
+
+    pub fn is_failed(&self) -> bool {
+        match self {
+            SubgraphHealth::Healthy => false,
+            SubgraphHealth::Unhealthy => true,
```

Comment:
While we don't have a proper UI design for the three health states, it seems preferable for the unhealthy state to be displayed as "Failed". So we can take the opportunity to change this behavior.

Reply:
Changed